docs/DeveloperGuide.md

```diff
@@ -409,10 +409,12 @@ _{Explain here how the data archiving feature will be implemented}_
 
 **Target user profile**:
 
-* logistics managers who have a need to manage a significant number of contacts
-responsible for logistical assets
-* prefer desktop apps over other types
-* can type fast and prefers typing to other forms of input
+* logistics managers 
+* has a need to manage a significant number of contacts and their respective logistical assets
+* has a need to organise all assets
```

Comment:
Perhaps this two lines can merged to be
+ has a need to keep track of a significant number of contacts and assets they are responsible for

Reply:
merged statements